src/reference/asciidoc/kafka.adoc

```diff
@@ -1889,6 +1889,272 @@ However, see the note at the beginning of this section; you can avoid using the
 
 IMPORTANT: If the recoverer fails (throws an exception), the record will be included in the seeks and recovery will be attempted again during the next delivery.
 
+[[container-props]]
+==== Listener Container Properties
+
+.`ContainerProperties` Properties
+[cols="6l,3,16", options="header"]
+|===
+| Property
+| Default
+| Description
+
+|ackCount
+|1
+|The number of records before committing pending offsets when the `ackMode` is `COUNT` or `COUNT_TIME`.
+
+|ackMode
+|BATCH
+|Controls how often offsets are committed - see <<committing-offsets>>.
+
+|ackOnError
+|`false`
+|[DEPRECATED in favor of `ErrorHandler.isAckAfterHandle()`]
+
+|ackTime
+|5000
+|The time in milliseconds after which pending offsets are committed when the `ackMode` is `TIME` or `COUNT_TIME`.
+
+|assignment
+CommitOption
+|LATEST_ONLY _NO_TX
+|Whether or not to commit the initial position on assignment; by default, the initial offset will only be committed if the `ConsumerConfig.AUTO_OFFSET_RESET_CONFIG` is `latest` and it won't run in a transaction even if there is a transaction manager present.
+See the javadocs for `ContainerProperties.AssignmentCommitOption` for more information about the available options.
+In 2.3 and 2.4, the default was `ALWAYS` to retain the previous default before it was configurable.
```

Comment:
Why should we mention an old behavior in this new docs?

Reply:
I was trying to make up for the fact that I failed to document the property in 2.3 😄